app/src/main/java/com/framgia/soundclound/screen/home/HomeViewModel.java

```diff
@@ -30,6 +39,9 @@ public void setHomeAdapter(HomeAdapter homeAdapter) {
 
     @Override
     public void onClick(Genre genres) {
-        // handle click item
+        Intent intent = new Intent(mContext, GenreDetailActivity.class);
+        intent.putExtra(Constant.EXTRA_GENRE, genres.getKeyname());
+        mContext.startActivity(intent);
```

Comment:
2.11 Argument in Activity and Fragment
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md